app/record-a-vaccination/2025/12/customer-experience-survey/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Our first customer experience survey
+date: 2025-12-19
+---
+
+On 26 November 2025, we emailed all users of our service, around 16,500 people, asking them to take part in a short customer experience survey. This post is about what we asked and what our users told us. 
+
+## Why a survey and why now
+
+Record a vaccination (RAVS) has been used to record vaccinations since September 2024. Over the past year we’ve done several rounds of in-person and remote user research and usability testing. 
+
+We’ve also had a permanent link to a feedback survey in the header of our service. This has provided useful insights but it’s not been widely used. In total, we’ve had just over 400 responses and they have not been evenly spread over time. 
```

Comment:
Should we also mention that this survey has a bias because people only answer it when they have some specific feedback to give?